tests/test_invoke.py

```diff
@@ -192,15 +192,15 @@ def create_fake_hmplugin(plugin_dir: Path) -> Path:
     return hmplugin_filename
 
 
-def test_install_plugin_to_user_folder(new_plugin_dir: Path, tmp_path: Path):
+def test_install_plugin(new_plugin_dir: Path, tmp_path: Path):
     fake_hmplugin = create_fake_hmplugin(new_plugin_dir)
     assert fake_hmplugin.is_file()
     fake_hmplugin_files = get_plugin_installed_files(new_plugin_dir)
 
     user_folder_env_var = "USERPROFILE" if sys.platform == "win32" else "HOME"
     os.environ[user_folder_env_var] = str(tmp_path)
 
-    subprocess.run([f"{invoke_cmd}", "install-plugin-to-user-folder"], env=os.environ)
+    subprocess.run([f"{invoke_cmd}", "install-plugin"], env=os.environ)
 
     install_dir = Path("~/.alfasim_plugins").expanduser()
     os.chdir(install_dir)
```

Comment:
In the future instead doing `os.chdir` and mutating `os.environ` prefer to use the fixture `monkeypatch` and do `monkeypatch.chdir` and `monkeypatch.setenv` and the fixture will restore the original global state affected by changing the cwd or the environment.


Also, when doing:
```
    os.environ[user_folder_env_var] = str(tmp_path)
    subprocess.run([f"{invoke_cmd}", "install-plugin"], env=os.environ)
```
you could do:
```
    env = {**os.environ, user_folder_env_var: str(tmp_path)}
    subprocess.run([f"{invoke_cmd}", "install-plugin"], env=env)
```
The same effect without tainting the global `os.environ` variable.

Reply:
Thanks for the tip. I've updated the code to use the monkeypatch.